<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Modelagem ArboBA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 75, 75);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameInicial">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>1021</width>
      <height>841</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tabButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>310</y>
       <width>200</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(130, 130, 130);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Tabular</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pltButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>410</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(130, 130, 130);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Gráfico</string>
     </property>
    </widget>
    <widget class="QPushButton" name="geoRefButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>699</y>
       <width>200</width>
       <height>61</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(130, 130, 130);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Georreferenciamento</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboAno">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>660</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPlot">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>620</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Automática - Casos por Município</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sturges - Casos por Município</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Automática - Casos por 100 mil habitantes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sturges - Casos por 100 mil habitantes</string>
      </property>
     </item>
    </widget>
    <widget class="QTableView" name="dataframe">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>700</width>
       <height>580</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>700</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Arbovirose</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboAno_3">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>260</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboAno_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>260</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>190</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Selecione os anos a serem compararados&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>550</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Selecione os critérios para georeferenciamento&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
